flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/config/AutoScalerOptions.java

```diff
@@ -98,6 +98,13 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
                     .withDescription(
                             "Max scale down factor. 1 means no limit on scale down, 0.6 means job can only be scaled down with 60% of the original parallelism.");
 
+    public static final ConfigOption<Double> MAX_SCALE_UP_FACTOR =
+            autoScalerConfig("scale-up.max-factor")
+                    .doubleType()
+                    .defaultValue(2.0)
+                    .withDescription(
+                            "Max scale up factor. 2.0 means job can only be scaled up with 200% of the original parallelism.");
```

Comment:
```suggestion
                            "Max scale up factor. 2.0 means job can only be scaled up with 200% of the current parallelism.");
```

Just to make it clear that it can double every time.